include/RevLoader.h

```diff
@@ -355,11 +348,8 @@ class RevLoader {
   /// Loads a 64bit Elf binary
   bool LoadElf64( char* MemBuf, size_t Size );
 
-  ///< Splits a string into tokens
-  void splitStr( const std::string& s, char c, std::vector<std::string>& v );
-
   ///< Breaks bulk writes into cache lines
-  bool WriteCacheLine( uint64_t Addr, size_t Len, void* Data );
+  bool WriteCacheLine( uint64_t Addr, size_t Len, const void* Data );
```

Comment:
Probably a stupid question but how does `const void*` actually work? No different than something like a const char*?

Reply:
Any pointer (to non-`volatile`) can be implicitly converted to `const void*`.

A pointer to `const T` cannot be implicitly converted to `void*` since it drops `const`, hence the reason for this change.

I was trying to write `s.c_str()` to write out a string, and got an error that its `const char*` could not be converted to `void*`.